core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -3068,6 +3141,14 @@ object Stream extends StreamLowPriority {
   def evalUnChunk[F[_], O](fo: F[Chunk[O]]): Stream[F, O] =
     fromFreeC(Algebra.eval(fo).flatMap(Algebra.output))
 
+  /** Like `eval`, but lifts a foldable structure. **/
+  def evals[F[_], S[_]: Foldable, O](fo: F[S[O]]): Stream[F, O] =
+    eval(fo).flatMap(_.foldMap(Stream.emit))
```

Comment:
I went with `foldMap`, but also considered `Stream.emits(foo.toList)`. Should I benchmark whether one would have a significant performance improvement over the other?